.github/workflows/lighthouse.yml

```diff
@@ -0,0 +1,117 @@
+# Security Notes
+# This workflow uses `pull_request_target`, so will run against all PRs automatically (without approval), be careful with allowing any user-provided code to be run here
+# Only selected Actions are allowed within this repository. Please refer to (https://github.com/nodejs/nodejs.org/settings/actions)
+# for the full list of available actions. If you want to add a new one, please reach out a maintainer with Admin permissions.
+# REVIEWERS, please always double-check security practices before merging a PR that contains Workflow changes!!
+# AUTHORS, please only use actions with explicit SHA references, and avoid using `@master` or `@main` references or `@version` tags.
+# MERGE QUEUE NOTE: This Workflow does not run on `merge_group` trigger, as this Workflow is not required for Merge Queue's
+
+name: Lighthouse
+
+on:
+  pull_request_target:
+    branches:
+      - main
+    types:
+      - labeled
+
+defaults:
+  run:
+    # This ensures that the working directory is the root of the repository
+    working-directory: ./
+
+permissions:
+  contents: read
+  actions: read
+  # This permission is required by `thollander/actions-comment-pull-request`
+  pull-requests: write
+
+jobs:
+  lighthouse-ci:
+    # We want to skip our lighthouse analysis on Dependabot PRs
+    if: startsWith(github.event.pull_request.head.ref, 'dependabot/') == false
```

Comment:
What about Crowdin PRs? Is it relevant to run Lighthouse there?

Reply:
I'd argue whereever we run a vercel deployment we run lighthouse

Reply:
Right, that's fair.